defaults/test_default.yaml

```diff
@@ -205,7 +205,7 @@ stress_image:
   nosqlbench: 'scylladb/hydra-loaders:nosqlbench-4.15.49'
   cassandra-stress: '' # default would be same version as scylla under test
   scylla-bench: 'scylladb/hydra-loaders:scylla-bench-v0.1.17'
-  gemini: 'scylladb/hydra-loaders:gemini-1.8.0'
+  gemini: 'scylladb/hydra-loaders:gemini-1.8.2'
```

Comment:
The built image has `v` in the tag name:
https://hub.docker.com/layers/scylladb/hydra-loaders/gemini-v1.8.2/images/sha256-f8f566f0c5845d6fabc517c29f72fbcf996e57feb546f598bd05355aec9ca910?context=explore

And yes, it is different than it was before without `v` -> https://hub.docker.com/layers/scylladb/hydra-loaders/gemini-1.8.0/images/sha256-da5b5b4b46c1e7da9ab7899b1e0c228d4be2d4923cd53380ec34b37f26a8d943?context=explore

So, need to fix one of the 2 places.

Reply:
there were more changes besides naming... I adapted SCT.